sympy/logic/boolalg.py

```diff
@@ -1745,6 +1905,24 @@ def _convert_to_varsPOS(maxterm, variables):
     return Or(*temp)
 
 
+def _convert_to_varsANF(term, variables):
```

Comment:
can you document what `term` and `variables` are here?

Reply:
`term` and `variables` are the same arguments as arguments in `_convert_to_varsSOP` and `_convert_to_varsANF`. Is it necessary to document it?

Reply:
done